Breaks `x86_64-darwin`, but not sure if that also affects `aarch64-darwin` as that has broken reverse dependencies:

## `nixpkgs-review` result for [#454486](https://github.com/NixOS/nixpkgs/pull/454486)

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 454486`
Commit: [`64760444d55949ecef7d9557d67ff39a3794e285`](https://github.com/NixOS/nixpkgs/commit/64760444d55949ecef7d9557d67ff39a3794e285) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/64760444d55949ecef7d9557d67ff39a3794e285..pull/454486/head))
Merge: [`8e960a0013385d81c6b3d7bae34da7e4c1801fdd`](https://github.com/NixOS/nixpkgs/commit/8e960a0013385d81c6b3d7bae34da7e4c1801fdd)

Logs: https://github.com/jopejoe1/nixpkgs-review-gha/actions/runs/18713591063


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>fritz-exporter</li>
    <li>fritz-exporter.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>fritz-exporter</li>
    <li>fritz-exporter.dist</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>fritz-exporter</li>
    <li>fritz-exporter.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>fritz-exporter</summary>
<pre>DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:742 WLANCapability._generateMetricValues fetching metrics for repeater-wohnzimmer: 0
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:734 WLANConfigurationInfo._generateMetricValues checking WLAN 1 (enabled: True) on repeater-wohnzimmer
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:742 WLANCapability._generateMetricValues fetching metrics for repeater-wohnzimmer: 1
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:734 WLANConfigurationInfo._generateMetricValues checking WLAN 2 (enabled: True) on repeater-wohnzimmer
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:742 WLANCapability._generateMetricValues fetching metrics for repeater-wohnzimmer: 2
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:734 WLANConfigurationInfo._generateMetricValues checking WLAN 3 (enabled: False) on repeater-wohnzimmer
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:83 Fetching HostInfo metrics for fritz.box: False
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:83 Fetching HostInfo metrics for repeater-wohnzimmer: False
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:83 Fetching HomeAutomation metrics for fritz.box: True
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:1217 Fetching home automation device information for index 0
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:1217 Fetching home automation device information for index 1
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:1223 Got IndexError for index 1, stopping
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:83 Fetching HomeAutomation metrics for repeater-wohnzimmer: True
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:1217 Fetching home automation device information for index 0
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:1217 Fetching home automation device information for index 1
DEBUG    fritzexporter.fritzcapability:fritzcapabilities.py:1223 Got IndexError for index 1, stopping
INFO     fritzexporter:__main__.py:118 Starting listener at 127.0.0.1:9787
=========================== short test summary info ============================
FAILED tests/test_main.py::Test_Main::test_valid_args_run_clean - PermissionError: [Errno 1] Operation not permitted
========================= 1 failed, 63 passed in 2.48s =========================</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>fritz-exporter</li>
    <li>fritz-exporter.dist</li>
  </ul>
</details>

--- a/pkgs/by-name/fr/fritz-exporter/package.nix
+++ b/pkgs/by-name/fr/fritz-exporter/package.nix
@@ -6,14 +6,14 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "fritz-exporter";
-  version = "2.5.2";
+  version = "2.6.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "pdreker";
     repo = "fritz_exporter";
     tag = "fritzexporter-v${version}";
-    hash = "sha256-xQLTI6b8X22aU6dj7Tmkzxn7vE4y8r/djUetG3Qg9Qw=";
+    hash = "sha256-m2jDQN6c3S4xDIrmRFdD+stwutBxcespLKZvxp1VC0I=";
   };
 
   postPatch = ''
@@ -42,6 +42,9 @@ python3.pkgs.buildPythonApplication rec {
   nativeCheckInputs = with python3.pkgs; [
     pytestCheckHook
   ];
+
+  # Required for tests
+  __darwinAllowLocalNetworking = true;
 
   meta = {
     changelog = "https://github.com/pdreker/fritz_exporter/blob/${src.tag}/CHANGELOG.md";

